src/Dfe.PlanTech.Infrastructure.SignIn/Services/DfePublicApiService.cs

```diff
@@ -13,21 +13,22 @@ public class DfePublicApiService : IDfePublicApi
 {
     private readonly IDfeSignInConfiguration _dfeSignInConfiguration;
     private readonly string _urlTemplate = "{0}/services/{1}/organisations/{2}/users/{3}";
+    private readonly HttpClient _httpClient;
 
 
-    public DfePublicApiService(IDfeSignInConfiguration configuration)
+    public DfePublicApiService(IDfeSignInConfiguration configuration, HttpClient httpClient)
     {
+        _httpClient = httpClient;
         _dfeSignInConfiguration = configuration;
     }
 
     public async Task<UserAccessToService?> GetUserAccessToService(string userId, string organisationId)
     {
         var url = string.Format(_urlTemplate, _dfeSignInConfiguration.APIServiceProxyUrl, _dfeSignInConfiguration.ClientId, organisationId, userId);
-        var httpClient = new HttpClient();
-        httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer",
+        _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer",
```

Comment:
Best practice would be to move this to the HttpClient setup in the DfeSignInSetup.cs; let me know if you have questions!